src/intervals/arithmetic/absmax.jl

```diff
@@ -0,0 +1,37 @@
+# This file is part of the IntervalArithmetic.jl package; MIT licensed
```

Comment:
I suggest that we add this *banner* to all files of the package... I just realized I didn't mention that before.

Reply:
Is this really useful? Going through all file to add it is no big deal, but I don't see what it achieves.

Reply:
I'm not sure if it is useful, but I don't think it's something you need to do in this PR. You have already done a lot and that can be added later